04/dbernhard/tests_if8.oat

```diff
@@ -0,0 +1,14 @@
+int program(int argc, string[] args) {
+    return function(0) + function(1) + function(2) + function(3);
+}
+
+int function(int x) {
+    if (x == 0) {
+        
```

Comment:
Same here and in the other if branches where no return statement appears

Reply:
Are here actually implicit returns?

Reply:
Oh sorry I didn't see the last statement